iambic/plugins/v0_1_0/aws/cloud_formation/utils.py

```diff
@@ -458,21 +460,35 @@ async def create_hub_account_stacks(
     if tags:
         additional_kwargs["Tags"] = tags
 
-    stack_created = await create_stack(
-        cf_client,
-        stack_name=stack_name,
-        template_body=get_iambic_hub_role_template_body(),
-        parameters=[
-            {"ParameterKey": "HubRoleName", "ParameterValue": hub_role_name},
-            {
-                "ParameterKey": "SpokeRoleName",
-                "ParameterValue": spoke_role_name,
-            },
-            {"ParameterKey": "AssumeAsArn", "ParameterValue": assume_as_arn},
-        ],
-        Capabilities=["CAPABILITY_NAMED_IAM"],
-        **additional_kwargs,
-    )
+    stack_created = False
+    try:
+        stack_created = await create_stack(
+            cf_client,
+            stack_name=stack_name,
+            template_body=get_iambic_hub_role_template_body(),
+            parameters=[
+                {"ParameterKey": "HubRoleName", "ParameterValue": hub_role_name},
+                {
+                    "ParameterKey": "SpokeRoleName",
+                    "ParameterValue": spoke_role_name,
+                },
+                {"ParameterKey": "AssumeAsArn", "ParameterValue": assume_as_arn},
+            ],
+            Capabilities=["CAPABILITY_NAMED_IAM"],
+            **additional_kwargs,
+        )
+    except ClientError as e:
+        if (
+            "You must enable organizations access to operate a service managed stack set"
+            in e.response.get("Message", "Unknown")
+        ):
+            log.info(
```

Comment:
Should this be log.info ? Seems like it should be log.exception or log.error if we're raising right afterwards.